tests/wpunit/Tribe/Tickets/Status/ManagerTest.php

```diff
@@ -61,15 +61,14 @@ private function make_instance() {
 	}
 
 	/**
-	 * todo active test once RSVP and Tribe Commerce Managers are created
 	 *
 	 * @test
 	 * @since TBD
 	 */
-	/*	public function it_has_manage_class_keys_for_rsvp_and_tribe_commerce() {
-			$this->assertArrayHasKey( 'rsvp', Manager::get_instance()->get_status_managers() );
-			$this->assertArrayHasKey( 'tribe-commerce', Manager::get_instance()->get_status_managers() );
-		}*/
+	public function it_has_manage_class_keys_for_rsvp_and_tribe_commerce() {
+		$this->assertArrayHasKey( 'rsvp', Manager::get_instance()->get_status_managers() );
+		//$this->assertArrayHasKey( 'tribe-commerce', Manager::get_instance()->get_status_managers() );
```

Comment:
Should this be uncommented?

Reply:
It will be with the Tribe Commerce PR :)